odl_video/settings.py

```diff
@@ -21,6 +21,14 @@
 
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
+try:
+    with open(f'{BASE_DIR}/.env') as envsettings:
+        for line in envsettings:
+            k, v = line.rstrip('\n').lstrip('export ').split('=', maxsplit=1)
+            os.environ.setdefault(k=v)
```

Comment:
This throws an error: `TypeError: setdefault() got an unexpected keyword argument 'k'`

This works: `os.environ.setdefault(k, v)`